ios/FluentUI/Vnext/Drawer/SlideOverPanel.swift

```diff
@@ -111,14 +164,15 @@ struct SlideOverPanelLeft_Previews: PreviewProvider {
     static var previews: some View {
         ZStack {
             MockBackgroundView()
-            MSFSlideOverPanel<MockContent>(slideOutPanelWidth: UIScreen.main.bounds.width,
-                                           actionOnBackgroundTap: nil,
-                                           content: MockContent(),
-                                           backgroundLayerOpacity: 0.5,
-                                           direction: .left,
-                                           isOpen: Binding.constant(true),
-                                           preferredContentOffset: Binding.constant(nil),
-                                           tokens: MSFDrawerTokens())
+            MSFSlideOverPanel<MockContent>(
+                percentTransition: Binding.constant(0),
```

Comment:
nit: move the first parameter to the line above and indent accordingly? (here and below too)